docs/API.md

```diff
@@ -269,8 +273,9 @@ Parameters.
 
 Each token takes the following fields for it's create method respectively:
 
-* sessionToken : data, uid, createdAt, uaBrowser, uaBrowserVersion, uaOS, uaOSVersion, uaDeviceType
-* keyFetchToken : authKey, uid, keyBundle, createdAt
+* sessionToken : data, uid, createdAt, uaBrowser, uaBrowserVersion, uaOS, uaOSVersion, uaDeviceType,
+                 tokenVerified, tokenVerificationId
+* keyFetchToken : authKey, uid, keyBundle, createdAt, tokenVerificationId
```

Comment:
Earlier I thought no, because we always want keyFetchTokens to be created unverified. But I just realised you're right, because the auth server isn't going to be verifying keyFetchTokens in the first cut. In that case, the code as implemented would end up with a bunch of keyFetchTokens that never get verified.

Will fix, thanks for pointing it out.